src/utils/reputation.ts

```diff
@@ -28,43 +29,58 @@ export async function getReputation(userID: UserID): Promise<number> {
             SUM(CASE WHEN "timeSubmitted" > 1596240000000 THEN "votes" ELSE 0 END) AS "votedSum",
             SUM(locked) AS "lockedSum",
             SUM(CASE WHEN "timeSubmitted" < ? AND "timeSubmitted" > 1596240000000 AND "votes" > 0 THEN 1 ELSE 0 END) AS "semiOldUpvotedSubmissions",
-            SUM(CASE WHEN "timeSubmitted" < ? AND "timeSubmitted" > 1596240000000 AND "votes" > 0 THEN 1 ELSE 0 END) AS "oldUpvotedSubmissions"
+            SUM(CASE WHEN "timeSubmitted" < ? AND "timeSubmitted" > 1596240000000 AND "votes" > 0 THEN 1 ELSE 0 END) AS "oldUpvotedSubmissions",
+            SUM(CASE WHEN "votes" > 0 
+                AND NOT EXISTS (
+                    SELECT * FROM "sponsorTimes" as c 
+                    WHERE c."videoID" = "a"."videoID" AND c."votes" > "a"."votes" LIMIT 1) 
```

Comment:
```suggestion
                    WHERE c."videoID" = "a"."videoID" AND (c."votes" > "a"."votes" OR  c."locked" > "a"."locked") AND c."category" = "a"."category" LIMIT 1) 
```

Check against other segments of this category, and check for locked segments if this segment is not locked,